core/src/main/java/bisq/core/dao/DaoFacade.java

```diff
@@ -530,8 +531,8 @@ public long getTotalAmountOfConfiscatedTxOutputs() {
 
 
     public List<Bond> getAllBonds() {
-        List<Bond> bonds = bondedReputationRepository.getActiveBonds();
-        bonds.addAll(bondedRolesRepository.getActiveBonds());
+        List<Bond> bonds = new ArrayList<>(bondedReputationRepository.getBonds());
```

Comment:
Not sure if that is ok, as the ProposalDisplay class uses that as well:
` confiscateBondComboBox.setItems(FXCollections.observableArrayList(daoFacade.getAllBonds()));`
and there we want only the active bonds.

Reply:
Also used by `daoFacade.getBondByLockupTxId` which is also used in ProposalDisplay.

Reply:
Yes, but it is needed there to display also old items correctly.
For the `setItems` I'm not sure as you do want to display the link if you choose to display the proposal, but there is no reason to have such an active proposal in the first place.

Anyway, in `getBondByLockupTxId` we want also non-active bonds so it is possible to see past proposals correctly.

Reply:
I agree but there seems to be no need for getActiveBonds() method for now.